website/integrations/services/terrakube/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Integrate Terrakube
+sidebar_label: Terrakube
+---
+
+# Terrakube
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is Terrakube
+
+> Terrakube is an open-source collaboration platform designed for managing remote Infrastructure-as-Code (IaC) operations with Terraform. It serves as a alternative to proprietary tools like Terraform Enterprise.
+>
+> -- https://terrakube.io/
+
+## Preparation
+
+The following placeholders are used in this guide:
+
+- `terrakube-dex.company` is the FQDN of the [Dex](https://dexidp.io/) container of the Terrakube install.
+- `authentik.company` is the FQDN of the authentik install.
+
+## authentik configuration
+
+1. From the Admin interface, navigate to **Applications** -> **Applications**.
+2. Use the wizard to create a new application and provider. During this process:
+    - Note the **Client ID**, **Client Secret**, and **slug** values because they will be required later.
+    - Set a `Strict` redirect URI to ` https://terrakube-dex.company/dex/callback`.
+    - Select any available signing key.
+
+## Terrakube configuration
+
+This guide assumes you have environment variables `$TERRAKUBE_OIDC_CLIENT_ID` and `$TERRAKUBE_OIDC_CLIENT_SECRET` set up. While you can hardcode values if your setup doesn’t support environment variables, doing so is not recommended for security reasons.
+
+1. **Locate the Dex Configuration File**  
+   Find the Dex configuration file, typically named `config.yaml` or `config.docker.yaml`. It’s usually located in the `/etc/dex` directory or the corresponding directory for a containerized setup.
+
+2. **Update the Dex Configuration**  
+   Open the configuration file and add the following block to define the Terrakube OIDC connector:
```

Comment:
```suggestion
   To define the Terrakube OIDC connector, open the configuration file and add the following block:
```

Reply:
Editor's pro tip: when documenting a step or a task, try to start with the actual task, then describe how/where to do it.